active/0012-systemd-integration.md

```diff
@@ -0,0 +1,299 @@
+# Integration with Systemd
+
+## Changelog
+
+* 2021-06-09: @zhongwencool Initial draft
+
+
+## Abstract
+
+Emqx currently uses a simple systemd forking type to start emqx in demon mode. The other benefits of systemd are not fully exploited, for example, systemd does not restart all emqx processes that exit abnormally (`kill GroupChildrenPid`). Systemd  just acts as a starter.
+
+## Motivation
+
+The benefits of [systemd](https://github.com/systemd/systemd) include, but are not limited to:
+
+- The ability to configure specific system parameters for service such as `LimitNOFILE`.
```

Comment:
This is already the case even in fork mode ?
if yes, maybe better to state it here.